client/src/pages/ChatsBlock.jsx

```diff
@@ -0,0 +1,300 @@
+import { useContext } from 'react';
+import { userService } from '../services/userService.js';
+import { Formik, Form, Field } from 'formik';
+import { Context } from '../context/context.jsx';
+import { useNavigate } from 'react-router-dom';
+import { ErrorHandler } from './ErrorHandler.jsx';
+
+function chatNameValidation(newChatName) {
+  if (!newChatName || newChatName.trim().length === 0) {
+    return 'Chat name required';
+  }
+
+  if (newChatName.trim().length < 2) {
+    return 'Chat name is too short';
+  }
+
+  if (newChatName.length > 30) {
+    return 'Less than 30 characters required';
+  }
+}
```

Comment:
The `chatNameValidation` function does not return anything when the chat name is valid. It would be better to explicitly return `undefined` or `null` at the end of the function to indicate that there are no validation errors.